tests/src/helpers.c

```diff
@@ -244,3 +244,16 @@ void mbedtls_param_failed( const char *failure_condition,
     }
 }
 #endif /* MBEDTLS_CHECK_PARAMS */
+
+#if defined(MBEDTLS_TEST_HOOKS)
+void mbedtls_test_err_add_check( int high, int low,
+                                 const char *file, int line )
+{
+    if ( high > -0x1000 || low < -0x007F )
+    {
+        mbedtls_fprintf( stderr, "\nIncorrect error code addition at %s:%d\n",
```

Comment:
It would be nicer to have different error messages for each condition (positive high, positive low, high contains low bits, low contains high bits, high is outside 16-bit range, low is outside 16-bit range).

Reply:
Done.